models/userModel.ts

```diff
@@ -20,6 +25,16 @@ const userSchema = new Schema<UserDocument, {}, Methods>({
     name: { type: String, required: true, trim: true },
     password: { type: String, required: true},
     role: { type: String, enum: ["admin", "user"], default: "user" },
+    nameTag: {
+        fullName: { type: String },
+        preferredName: { type: String },
+        pronouns: { type: String },
+        disclosure: { type: String },
+        visible: { type: Boolean }
```

Comment:
Do we really want to store the visibility bit in the DB?
I would have thought that starting a new meeting would start with the nametag not visible. Am I wrong there?

Reply:
I guess it depends on what we want the default behavior to be. Do we want it to default to _on_, _off_, or the user's last setting? @Shaomei any thoughts?